db/db_impl_write.cc

```diff
@@ -247,6 +247,7 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
         last_sequence = versions_->FetchAddLastToBeWrittenSequence(total_count);
       }
     }
+    assert(last_sequence != kMaxSequenceNumber);
```

Comment:
The warning was that in line 251, last_sequence  can be uninitialized. I didn't see it possible but I initialized it and added an assert anyway.